### PR TITLE
Update en_GB.ts

### PR DESCRIPTION
**Pull request by Gooochy**
I have updated it so it shows why and where it is required

### CODE REVIEW

File: src/i18n/en_GB.ts [hunk 1]
```diff
@@ -485,7 +485,7 @@ export default {
             schooling: 'Rescue - SORT Training',
             shownSchooling: 'SORT',
             special:
-                'You can buy 1 Mass Casualty Equipment for every 20 ambulance stations (respectively 15 with premium account).',
+                'You can buy 1 Mass Casualty Equipment for every 20 ambulance stations (respectively 15 with premium account). It is required for missions that can spawn with over 30 paitents',
```
Comment (JRH-1997) [inline src/i18n/en_GB.ts]: paitents => Patients

Comment (Suchty112) [inline src/i18n/en_GB.ts:488]: patients*

File: src/i18n/en_GB.ts [hunk 2]
```diff
@@ -496,7 +496,7 @@ export default {
             maxPersonnel: 1,
             schooling: 'Rescue - Ambulance Officer',
             shownSchooling: 'Ambulance Officer',
-            special: 'Required once you have built 15 Rescue stations',
+            special: 'Required once you have built 15 Rescue stations. It is required for missions that can spawn with over 20 paitents to help command the scene it is a dual unit betweeen a OTL and a Ambulance Officer Requirements',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:499]: patients*

**Review by Suchty112 — CHANGES_REQUESTED** [review]

File: src/i18n/en_GB.ts (hunk 2 above)
Comment (JRH-1997) [inline src/i18n/en_GB.ts]: Same with patients 
betweeen => between
an OTL and an Ambulance

Comment (JRH-1997) [reply src/i18n/en_GB.ts]: Is it needed to say that it is an dual unit? That info is not at the other dual units

Comment (Gooochy) [reply src/i18n/en_GB.ts]: It is I have done it for other dual units and it is not a ambulance

Comment (JRH-1997) [reply src/i18n/en_GB.ts]: I don't mean it's acting as ambulance. But you wrote `a` ambulance, and `a` OTL. It should be `an` in both cases

File: src/i18n/en_GB.ts [hunk 3]
```diff
@@ -496,7 +496,7 @@ export default {
             maxPersonnel: 1,
             schooling: 'Rescue - Ambulance Officer',
             shownSchooling: 'Ambulance Officer',
-            special: 'Required once you have built 15 Rescue stations',
+            special: 'Required once you have built 15 Rescue stations. It is required for missions that can spawn with over 20 patients to help command the scene it is a dual unit between an OTL and an Ambulance Officer Requirements',
```
Comment (jxn-30) [inline src/i18n/en_GB.ts:499]: don't you want to split your added sentence up into two sentences?

Comment (Gooochy) [reply src/i18n/en_GB.ts:499]: Probably

**Review by Suchty112 — APPROVED** [review]

**Pull request merged**